src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="notifier" class="Symfony\Component\Notifier\Notifier">
+            <argument type="tagged_locator" tag="notifier.channel" index-by="channel" />
+            <argument type="service" id="notifier.channel_policy" on-invalid="ignore" />
+            <argument type="service" id="messenger.default_bus" on-invalid="ignore" />
+        </service>
+        <service id="Symfony\Component\Notifier\NotifierInterface" alias="notifier" />
+
+        <service id="notifier.channel_policy" class="Symfony\Component\Notifier\Channel\ChannelPolicy">
+            <argument type="collection" /> <!-- policy -->
+        </service>
+
+        <service id="notifier.browser_channel" class="Symfony\Component\Notifier\Channel\BrowserChannel">
```

Comment:
@fabpot some of my comments were resolved by pushing new version, but there are no changes related. Please take this into account and let me know if you resolved it on your own because you don't agree.